src/app/lib/config/services/cymrufyw.js

```diff
@@ -39,6 +39,7 @@ export const service = {
     datetimeLocale: `cy`,
     service: 'cymrufyw',
     serviceName: 'Cymru Fyw',
+    languageName: '',
```

Comment:
Is the default 'English' if this value is blank?

Reply:
not sure. It wasn't in the list. any idea @jimjohnsonrollings ?

Reply:
We don't serve any Cymru Fyw pages, but happy for it to have the languageName = `Welsh` if we need one

Reply:
If we need a value for Naidheachdan too, we should use `Scottish Gaelic`

Reply:
Thanks Jim.